libnavui-base/src/main/java/com/mapbox/navigation/ui/base/internal/model/route/RouteConstants.kt

```diff
@@ -110,6 +111,12 @@ object RouteConstants {
     @ColorInt
     val MANEUVER_ARROW_BORDER_COLOR = Color.parseColor("#2D3F53")
 
+    @ColorInt
+    val ROUTE_CLOSURE_COLOR = Color.parseColor("#000000")
```

Comment:
Do we want to default to `#000000`? 🤔 

cc @d-prukop

Reply:
We can follow up in a separate ticket. Could you cut a ticket when you have a chance @cafesilencio? 🙏